_site/index.html

```diff
@@ -0,0 +1,260 @@
+<!doctype html>
+<!--
+  Minimal Mistakes Jekyll Theme 4.13.0 by Michael Rose
+  Copyright 2013-2018 Michael Rose - mademistakes.com | @mmistakes
+  Free for personal and commercial use under the MIT license
+  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE.txt
+-->
+<html lang="en" class="no-js">
+  <head>
+    <meta charset="utf-8">
+
+<!-- begin _includes/seo.html --><title>Azure AI Gallery</title>
+<meta name="description" content="">
+
+
+
+<meta property="og:type" content="website">
+<meta property="og:locale" content="en_US">
+<meta property="og:site_name" content="Azure AI Gallery">
+<meta property="og:title" content="Azure AI Gallery">
+<meta property="og:url" content="http://localhost:4000/">
```

Comment:
What's this port 4000 stuff for? Looks like it comes from _site/feed.xml. Sorry I am not familiar with the framework but a non-standard port in would-be-production site content makes me nervous :)

Reply:
This would go along with the 'not including _site', 4000 is just the port I've been previewing on locally, so I think it was added for that reason. I will verify